doc/src/sgml/xfunc.sgml

```diff
@@ -1804,17 +1800,26 @@ ERROR:  could not determine polymorphic type because input has type unknown
     </para>
 
     <para>
+<!--
      With <function>make_array</function> declared as above, you must
      provide two arguments that are of exactly the same data type; the
      system will not attempt to resolve any type differences.  Thus for
      example this does not work:
+-->
+上記のように<function>make_array</function>を宣言した場合、まったく同じデータ型の2つの引数を指定する必要があります。
+システムは型の違いを解決しようとしません。  
+したがって、例えばこれはうまくいきません。
 <screen>
 SELECT make_array(1, 2.5) AS numericarray;
 ERROR:  function make_array(integer, numeric) does not exist
 </screen>
+<!--
      An alternative approach is to use the <quote>common</quote> family of
      polymorphic types, which allows the system to try to identify a
      suitable common type:
+-->
+別の方法として、<quote>common</quote>ファミリーの多型タイプを使用する方法があります。
```

Comment:
polymorphic type は上では多様型と訳していますが、ここで変えているのは意図的なものでしょうか。

Reply:
たぶんtypoです。誤って消してしまった様です。

Reply:
型タイプ だと typeを2回訳しているように見えますが、、、、
直後に出てくる「共通タイプ」と合わせようという意図があったりしますか。

Reply:
すみません、中途半端に出してしまってました。ここは主な説明があるextend.sgmlが「 \<quote\>common\<\/quote\> family of polymorphic types」をどう訳すかを見たいところでした。

これまで「family」を「族」としてきたので、前例に沿うと\<quote\>共通\<\/quote\>族の多様型になるでしょうか？
そうすると

>  An alternative approach is to use the  \<quote\>common\<\/quote\> family of polymorphic types, which allows the system to try to identify a  suitable common type:

\<quote\>共通\<\/quote\>族の多様型を使用する方法があります。
これにより、システムは適切な共通型を特定できます。

次も
> Because the rules for common type resolution default to choosing  type \<type\>text\<\/type\> when all inputs are of unknown types, this  also works:

すべての入力が未知の型である場合、共通の型を解決するルールはデフォルトで\<type\>text\<\/type\>型を選択するので、これも動作します。

にしようと思います。

Reply:
はい、私的にもそれがいいと思います。

extend.sgmlを見ようかと思ったのですが、まだ担当者なしなんですね、、、orz

Reply:
ありがとうございます。とりあえずはこれで修正します。